### PR TITLE
feat: support OPBNB cross-chain

### PR DESCRIPTION
**Pull request by alexgao001**
### Description

Modify the current relayer with minimal change to enable crosschain to OPBNB

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes:
* add each change in a bullet point here
* ...

### CODE REVIEW

File: listener/bsc_listener.go [hunk 1]
```diff
@@ -68,7 +68,13 @@ func (l *BSCListener) poll() error {
 		if nextHeight <= latestPolledBlockHeight {
 			nextHeight = latestPolledBlockHeight + 1
 		}
-		latestBlockHeight, err := l.bscExecutor.GetLatestFinalizedBlockHeightWithRetry()
+		var latestBlockHeight uint64
+		if l.isOpCrossChain() {
+			// currently Get finalized block is not support by OPBNB yet
+			latestBlockHeight, err = l.bscExecutor.GetLatestBlockHeightWithRetry()
```
Comment (yutianwu) [inline listener/bsc_listener.go:74]: do  we need to add more blocks for finality of opbnb

Comment (alexgao001) [reply listener/bsc_listener.go:74]: This is done is the voteProcessor, num of blocks will be waited before broadcasting vote

Comment (zjubfd) [reply listener/bsc_listener.go:74]: for opbnb, we consider it as one block finality.

**Review by anewt95 — COMMENTED** [review]
****

**Review by anewt95 — APPROVED** [review]

**Review by yutianwu — APPROVED** [review]

**Review by zjubfd — APPROVED** [review]

**Review by anewt95 — COMMENTED** [review]
$

**Pull request merged**